ui-components/dialogcomponents.md

```diff
@@ -330,6 +330,24 @@ An **input** is a composite component that renders a label and a single line tex
 }
 ```
 
+#### inputMode
```

Comment:
This looks to be incorrect, as it's not a new component (which is what #### is showing on this page), but instead a new property for the input comment. We don't really have a nice way of documenting the properties of components on this page by the looks of things :disappointed: